include/fastrtps/qos/ParameterTypes.h

```diff
@@ -808,6 +808,20 @@ class ParameterPropertyList_t : public Parameter_t
         properties.assign(parameter_properties.properties.begin(), parameter_properties.properties.end());
     }
 
+    ParameterPropertyList_t& operator=(const ParameterPropertyList_t& pl)
+    {
+       Parameter_t::operator=(pl);
+       properties = pl.properties;
+       return *this;
+    }
+
+    ParameterPropertyList_t& operator=(ParameterPropertyList_t&& pl)
+    {
+       Parameter_t::operator=(std::move(pl));
+       properties = std::move(pl.properties);
+       return *this;
+    }
+
```

Comment:
Uncrustify

Reply:
👍

Reply:
Why not just default and let the compiler handle the implementation?

Reply:
That's C++ 11. Lets try it!!! 😄

Reply:
Yes, that is C++11 but std::move is also C++11 ;-)